MovieApp/LoginViewController.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  LoginViewController.swift
+//  MovieApp
+//
+//  Created by endava-bootcamp on 02.04.2023..
+//
+
+import Foundation
+import PureLayout
+import UIKit
+
+
+class LoginViewController: UIViewController {
+    
+    private var signInLabel: UILabel!
+    private var emailLabel: UILabel!
+    private var passwordLabel: UILabel!
+    private var emailTextField: UITextField!
+    private var passwordTextField: UITextField!
+    private var signInButton: UIButton!
+    
+    private let backgroundColor = UIColor(red: 0.04, green: 0.15, blue: 0.25, alpha: 1.0)
+    //another way of changing backgorund color would be  view.backgroundColor = ... inside viewDidLoad()
+    init() {
+        super.init(nibName: nil, bundle: nil)
+        view.backgroundColor = self.backgroundColor
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        buildViews()
+        
+    }
+    
+    private func buildViews() {
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+    
+    private func createViews() {
+        signInLabel = UILabel()
+        emailLabel = UILabel()
+        passwordLabel = UILabel()
+        emailTextField = UITextField()
+        passwordTextField = UITextField()
+        signInButton = UIButton()
+        styleViews()
```

Comment:
Why do you call `styleViews` in `buildViews` method as well as in `createViews` method?